flexmeasures/data/models/time_series.py

```diff
@@ -25,10 +25,17 @@
 class Sensor(db.Model, tb.SensorDBMixin):
     """A sensor measures events. """
 
-    def __init__(self, name: str, **kwargs):
-        tb.SensorDBMixin.__init__(self, name, **kwargs)
-        tb_utils.remove_class_init_kwargs(tb.SensorDBMixin, kwargs)
-        db.Model.__init__(self, **kwargs)
+    generic_asset_id = db.Column(
+        db.Integer, db.ForeignKey("generic_asset.id"), nullable=False
+    )
+    generic_asset = db.relationship(
+        "GenericAsset",
+        foreign_keys=[generic_asset_id],
+        backref=db.backref("sensors", lazy=True),
+    )
+
+    def __init__(self, **kwargs):
+        super(Sensor, self).__init__(**kwargs)
```

Comment:
Why is `tb_utils.remove_class_init_kwargs(tb.SensorDBMixin, kwargs)` not necessary any more here?

Reply:
It actually does still seem necessary. I checked and this way `tb.SensorDBMixin` is not being called. I have a hunch that we are still not setting up our mixins correctly, because I think we should be able to work with that `super` function instead.